internal/commands/objectstorage/show.go

```diff
@@ -0,0 +1,137 @@
+package objectstorage
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/format"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/labels"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/ui"
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud/request"
+)
+
+// ShowCommand creates the "objectstorage show" command
+func ShowCommand() commands.Command {
+	return &showCommand{
+		BaseCommand: commands.New(
+			"show",
+			"Show Managed object storage service details",
+			"upctl objectstorage show 55199a44-4751-4e27-9394-7c7661910be8",
+		),
+	}
+}
+
+type showCommand struct {
+	*commands.BaseCommand
```

Comment:
`show` and `delete` commands could have completion and resolver implementations, but those can also be added in later PR. We might also want to wait until multiple object-storages are supported per region before implementing those 🤔